catkin_virtualenv/src/dh_virtualenv/deployment.py

```diff
@@ -71,8 +75,8 @@ def __init__(self,
 
         self.preinstall = preinstall
         self.upgrade_pip = upgrade_pip
+        self.pip_version = pip_version
```

Comment:
Should this have a `# (pbovbel): allow pinning the pip version`?

Reply:
Done